iceoryx_binding_c/include/iceoryx_binding_c/types.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IOX_BINDING_C_TYPES_H_
+#define IOX_BINDING_C_TYPES_H_
+
+enum iox_SubscribeState
+{
+    SubscribeState_NOT_SUBSCRIBED = 0,
+    SubscribeState_SUBSCRIBE_REQUESTED,
+    SubscribeState_SUBSCRIBED,
+    SubscribeState_UNSUBSCRIBE_REQUESTED,
+    SubscribeState_WAIT_FOR_OFFER
+};
+
+enum iox_popo_ChunkReceiveError
```

Comment:
since this also has "_SUCCESS" shouldn't this be a `iox_popo_ChunkReceiveResult`?

Reply:
done